Mlem/Views/Tabs/Posts/Components/Comment/Components/Comment Item.swift

```diff
@@ -70,47 +70,12 @@ struct CommentItem: View
             HStack(spacing: 12)
             {
                 #warning("TODO: Add post rating")
-                HStack
-                {
-                    HStack(alignment: .center, spacing: 2) {
-                        Image(systemName: "arrow.up")
-                        
-                        Text(String(localCommentScore ?? hierarchicalComment.commentView.counts.score))
-                    }
-                    .foregroundColor(upvoteColor)
-                    .onTapGesture {
-                        Task(priority: .userInitiated) {
-                            try await rate(hierarchicalComment, operation: .upvote)
-                        }
-                    }
-                    
-                    Image(systemName: "arrow.down")
-                        .foregroundColor(downvoteColor)
-                        .onTapGesture {
-                            Task(priority: .userInitiated) {
-                                try await rate(hierarchicalComment, operation: .downvote)
-                            }
-                        }
-                }
-                .accessibilityElement(children: .ignore)
```

Comment:
note: the accessibility here is already applied to the `VoteComplex` view that is now used here